command_manager.gd

```diff
@@ -110,6 +110,13 @@ func _connect_command_signals(command:Command) -> void:
 		command.command_finished.connect(_on_command_finished.bind(command), CONNECT_ONE_SHOT)
 
 
+func _disconnect_command_signals(command:Command) -> void:
```

Comment:
Commands are connected with `CONNECT_ONE_SHOT` flag, where this would be needed?